base/src/main/java/org/aya/concrete/stmt/QualifiedID.java

```diff
@@ -38,10 +38,6 @@ public boolean isUnqualified() {
     return join(ids());
   }
 
-  public @NotNull ModulePath.Qualified asModulePath() {
```

Comment:
I think I need this method.

Reply:
In general we should eliminate it. Where do you need it?

Reply:
Forget what I said 🥲